cmd/server.go

```diff
@@ -156,6 +160,7 @@ const (
 	DefaultExecutableName               = "atlantis"
 	DefaultMarkdownTemplateOverridesDir = "~/.markdown_templates"
 	DefaultGHHostname                   = "github.com"
+	DefaultGiteaBaseURL                 = "https://cloud.gitea.com"
```

Comment:
I think this URL is wrong, the correct one is: https://gitea.com/ (the index is a redirect to about.gitea.com, but the Gittea instance is on gitea.com)

You might also consider using the URL https://codeberg.org/

Reply:
Corrected. I opted for gitea.com and not something like codeberg since that's more logical IMHO.